barricade-annotations/src/main/java/com/mutualmobile/barricade/response/BarricadeResponseSet.java

```diff
@@ -10,9 +10,11 @@
 public class BarricadeResponseSet {
   public List<BarricadeResponse> responses;
   public int defaultIndex;
```

Comment:
I still feel defaultIndex doesn't convey that it is the current index that's in use. It makes it a little better that the next variable is originalDefaultIndex since it kinda hints that defaultIndex is dynamically variable. Also I think it will help if we can make the originalDefaultIndex final